mteb/tasks/BitextMining/multilingual/TedTalksBitextMining.py

```diff
@@ -0,0 +1,178 @@
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        n_samples={"train": _N},
+        avg_character_length={},
+        bibtex_citation="""
+@inproceedings{qi-etal-2018-pre,
+    title = "When and Why Are Pre-Trained Word Embeddings Useful for Neural Machine Translation?",
+    author = "Qi, Ye  and
+      Sachan, Devendra  and
+      Felix, Matthieu  and
+      Padmanabhan, Sarguna  and
+      Neubig, Graham",
+    editor = "Walker, Marilyn  and
+      Ji, Heng  and
+      Stent, Amanda",
+    booktitle = "Proceedings of the 2018 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 2 (Short Papers)",
+    month = jun,
+    year = "2018",
+    address = "New Orleans, Louisiana",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/N18-2084",
+    doi = "10.18653/v1/N18-2084",
+    pages = "529--535",
+    abstract = "The performance of Neural Machine Translation (NMT) systems often suffers in low-resource scenarios where sufficiently large-scale parallel corpora cannot be obtained. Pre-trained word embeddings have proven to be invaluable for improving performance in natural language analysis tasks, which often suffer from paucity of data. However, their utility for NMT has not been extensively explored. In this work, we perform five sets of experiments that analyze when we can expect pre-trained word embeddings to help in NMT tasks. We show that such embeddings can be surprisingly effective in some cases {--} providing gains of up to 20 BLEU points in the most favorable setting.",
+}
+""",
+    )
+
+    def load_data(self, **kwargs: Any) -> None:
+        """
+        Load dataset from HuggingFace hub
+        """
+        if self.data_loaded:
+            return
+        self.dataset = {}
+        for lang in self.langs:
+            self.metadata_dict["dataset"]["path"] = "davidstap/ted_talks"
+            self.dataset[lang] = DatasetDict(
+                {
+                    "train": load_dataset(
+                        name="_".join(
+                            [_ISO6393_to_ISO6391[l] for l in lang.split("-")]
+                        ),
+                        split=f"train[:{_N}]",
+                        **self.metadata_dict["dataset"],
+                    )
+                }
+            )
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def dataset_transform(self) -> None:
+        for lang in self.langs:
+            l1, l2 = lang.split("-")
+            for split in _SPLIT:
+                self.dataset[lang][split] = self.dataset[lang][split].rename_column(
+                    _ISO6393_to_ISO6391[l1], "sentence1"
+                )
+                self.dataset[lang][split] = self.dataset[lang][split].rename_column(
+                    _ISO6393_to_ISO6391[l2], "sentence2"
+                )
```

Comment:
I think you can do this instead:
```suggestion
                self.dataset[lang][split] = self.dataset[lang][split].rename_columns({
                    _ISO6393_to_ISO6391[l1]: "sentence1",
                    _ISO6393_to_ISO6391[l2]: "sentence2"
                })
```
See docs here: https://huggingface.co/docs/datasets/v2.18.0/en/package_reference/main_classes#datasets.Dataset.rename_columns

Reply:
Actually you can even remove the for loop for iterating on splits and do:
```suggestion
                self.dataset[lang] = self.dataset[lang].rename_column({
                    _ISO6393_to_ISO6391[l1]: "sentence1",
                    _ISO6393_to_ISO6391[l2]: "sentence2"
                })
```
It will rename columns for all the splits.

Reply:
Thanks that's a good suggestion! I accidentally included TedTalks file (form #386) here in this NTREX PR. I removed it now, so this code change is not relevant for this PR. I'll make this change at #386 though :)